nexus-repository-composer/src/main/java/org/sonatype/nexus/repository/composer/internal/ComposerProxyFacetImpl.java

```diff
@@ -203,7 +203,6 @@ private String getZipballUrl(final Context context) {
       try {
         String path = "/" + buildPackagePathForDevVersions(vendor, project);
         Payload payload = getPackagePayload(context, path);
-        String url = composerJsonProcessor.getDistUrlFromPackage(vendor, project, version, payload);
```

Comment:
was this maybe the only issue we had, because `payload` was not check beforehand or do we need the part with `package` and the `provider` path?

Reply:
Just some cleanup...